release/models/acl/openconfig-acl.yang

```diff
@@ -830,6 +838,61 @@ module openconfig-acl {
     }
   }
 
+  grouping prefix-list-config {
+     leaf name {
+        type string;
+        description
+           "Name of the prefix list to
+            be used in the matches";
+      }
+      leaf description {
+        type string;
+        description
+          "Prefix list description";
+      }
+      leaf-list prefix {
+         type oc-inet:ip-prefix;
+         description
+           "List of prefixes to be used in match
+           conditions";
+      }
+    }
+
+ grouping prefix-list-top {
+    description
+      "Configuration of Prefix Lists";
+    container prefix-sets {
+        description
+          "Container to hold list of IPv4 or IPv6
+           prefixes which are matched as part of a policy.";
+        list ipv4-prefix-set {
```

Comment:
Should this just be called `prefix-set` since it says above that it can be IPv4 or IPv6?

Reply:
Done, it is now called prefix-set with the rigth type (ipv4-prefix or ipv6-prefix) depending